hearts/views.py

```diff
@@ -32,16 +25,39 @@ class NewPlayerFormView(FormView):
     success_url = reverse_lazy('home')
 
     def form_valid(self, form: NewPlayerForm) -> HttpResponseRedirect:
-        player = self.create_player(form.name)
-        # TODO: Set cookie with player id.
-        return super().form_valid(form)
+        response = super().form_valid(form)
+        player = self.create_player(form.cleaned_data['name'])
+        response.set_cookie(PLAYER_ID_COOKIE_KEY, player.id)
+        return response
 
-    def create_player(self, name: str) -> Player:
+    @staticmethod
+    def create_player(name: str) -> Player:
         """
         Create a new Player.
         """
+        player = Player.objects.create(name=name)
+        return player
+
+
+@method_decorator(require_player, name='dispatch')
+class HomeTemplateView(TemplateView):
+    """
+    Home page.
+
+    Should direct user to server browser or create new game.
+    """
+
+    template_name = 'hearts/home.html'
+
+    def get_context_data(self, **kwargs) -> dict:
+        context = super().get_context_data(**kwargs)
+        context.update({
+            'player': self.request.player,
```

Comment:
The important part to note in this PR is right here. We can now access this new `player` attribute on the `request` object in all views that are decorated with the `require_player` decorator.